app/exporters/manual_section_publishing_api_exporter.rb

```diff
@@ -53,6 +45,50 @@ def exportable_attributes
     }
   end
 
+  def details
+    {
+      body: [
+        {
+          content_type: "text/govspeak",
+          content: document.attributes.fetch(:body)
+        },
+        {
+          content_type: "text/html",
+          content: rendered_document_attributes.fetch(:body)
+        }
+      ],
+      manual: {
+        base_path: "/#{manual.attributes.fetch(:slug)}",
+      },
+      organisations: [
+        organisation_info
+      ],
+    }.tap do |details_hash|
+      details_hash[:attachments] = attachments if document.attachments.present?
+    end
+  end
+
+  def attachments
+    document.attachments.map{|attachment| attachment_json_builder(attachment.attributes) }
+  end
+
+  def build_content_type(file_url)
+    return unless file_url
+    extname = File.extname(file_url).delete(".")
+    "application/#{extname}"
+  end
+
+  def attachment_json_builder(attributes)
```

Comment:
Is there a way to remove the duplication of these methods? It might not be worth the effort though if this is going to be replaced soon?